lang/en.json

```diff
@@ -1,4 +1,6 @@
 {
+  "audio player": "audio player",
+  "video player": "video player",
```

Comment:
Should these be title-cased for consistency as long as we're pushing toward a major release?